source/js/components/join/join.jsx

```diff
@@ -133,6 +133,20 @@ export default class JoinUs extends React.Component {
     });
   }
 
+  renderGAEventLabel() {
+    let formPosition = this.props.formPosition;
+    switch (formPosition) {
```

Comment:
Since this is a simple if/else, we can simplify switch statement into

```jsx
let gaLabel = `Signup submitted`;

if (formPosition) {
  gaLabel = `${label} from ${formPosition}`;
}
```

(You can merge this further into one line but I find that code hard to read. That's why I separate the `if` statement out)

Reply:
Note that

We don't have to worry too much on our end when it comes to letter case in event labels. GA does its magic to turn event labels into the letter case style it likes.
e.g.,


<img width="200" alt="image" src="https://user-images.githubusercontent.com/2896608/63619120-a67b8f00-c5a2-11e9-82f8-895838d40647.png">